futurex_openedx_extensions/helpers/exceptions.py

```diff
@@ -58,3 +63,13 @@ def __init__(self, code: FXExceptionCodes | int, message: str) -> None:
                 self.code = FXExceptionCodes.UNKNOWN_ERROR.value
         else:
             self.code = FXExceptionCodes.UNKNOWN_ERROR.value
+
+
+def fx_exception_handler(exc: Exception) -> Response | None:
+    """Handle exception and return response with error details"""
+    if isinstance(exc, FXCodedException):
+        return Response(
+            error_details_to_dictionary(reason=str(exc)),
+            status=status.HTTP_400_BAD_REQUEST
+        )
+    return None
```

Comment:
no need for a stand-alone function in my opinion. Please move the code to `handle_exception` method